### PR TITLE
ESSNTL-4006 fix double logging

### PR DESCRIPTION
**Pull request by r14chandra**

### CODE REVIEW

File: src/puptoo/upload.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import io
 import json
+import logging
 from minio import Minio
 from .utils import config, puptoo_logging
 
-logger = puptoo_logging.initialize_logging()
+logger = logging.getLogger(config.APP_NAME)
```
Comment (kgaikwad) [inline src/puptoo/upload.py:7]: Inside method `initialize_logging`, it handles cloudwatch related config that allows to app to publish logs to cloudwatch. Is there any other place where it is already giving call to this method on service start?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @r14chandra  for change. Added two inline comments.

File: src/puptoo/upload.py (hunk 1 above)
Comment (r14chandra) [reply src/puptoo/upload.py:7]: By mistake, I initialized the log here again which has already been done in [app.py](https://github.com/RedHatInsights/insights-puptoo/blob/bd41ff10f0c56183fedcaaa594396d6fc6965f6c/src/puptoo/app.py#L26) file.

Comment (r14chandra) [reply src/puptoo/upload.py:7]: > Inside method `initialize_logging`, it handles cloudwatch related config that allows to app to publish logs to cloudwatch. Is there any other place where it is already giving call to this method on service start?

It handles clowder+cloudwatch related logs.

**Review by kgaikwad — APPROVED** [review]
Thank you @r14chandra for adding inputs to my queries. Code change looks good to me.

**Pull request merged**